turbo.json

```diff
@@ -10,20 +10,12 @@
     "clean": {
       "cache": true
     },
-    "storybook-build": {
-      "outputs": [
-        "apps/pie-storybook/build/**"
-      ],
-      "dependsOn": [
-        "^build"
-      ]
-    },
```

Comment:
When it builds Storybook on CI does it build the components beforehand?

Reply:
Yep, as The `storybook-build` has now been renamed to `build` it's incorporated as part of that.

Turborepo will see that `pie-storybook` has a devDependency on `pie-button`, and will ensure that the dependants are built first, before attempting to build `pie-storybook` 👍🏼

Reply:
> Turborepo will see that pie-storybook has a devDependency on pie-button, and will ensure that the dependants are built first, before attempting to build pie-storybook 👍🏼

That might not be the correct approach under the premise of having an agnostic SB setup as Ellis asked us yesterday.

In such context SB should not have component dependencies.